evaluate/repository_test.go

```diff
@@ -314,16 +427,6 @@ func TestRepositoryLoadConfiguration(t *testing.T) {
 		TestDataPath:   filepath.Join("..", "testdata"),
 		RepositoryPath: filepath.Join("golang", "plain"),
 
-		MutationBefore: func(t *testing.T, repositoryPath string) {
-			configuration := bytesutil.StringTrimIndentations(`
-				{
-					"tasks": [
-						"write-tests"
-					]
-				}
-			`)
-			assert.NoError(t, os.WriteFile(filepath.Join(repositoryPath, "repository.json"), []byte(configuration), 0600))
-		},
```

Comment:
I think this needs to go into the previous commit where you added the config to the plain directory!